### DISCUSSION TITLE
OTB configs / results don't match

### DISCUSSION THREAD

**Discussion by amoskalev** [STVIR/pysot#423] [Reproduce]
Hi, 

I trained models with **siamrpn_r50_l234_dwxcorr_8gpu / siamrpn_r50_l234_dwxcorr_otb_8gpu** configs and tested on OTB. The highest score I got after hyper-parameter tuning is **0.687 / 0.676** resp. 

I trained siamrpn_r50_l234_dwxcorr_8gpu on 1 1080Ti and siamrpn_r50_l234_dwxcorr_otb_8gpu on 4 1080Ti's. 

I am a bit confused as 0.696 is reported in Model zoo for OTB. Did I miss something during the training? Is training results are dependent on # of gpus?

**Comment by ZhiyuanChen** [comment]
Have you adjusted batch size, learning rate, etc.?

**Comment by amoskalev** [comment]
@zyc-ai No, I didn't modify configs

**Comment by ZhiyuanChen** [comment]
> @zyc-ai No, I didn't modify configs

Then you probably should do it.

**Comment by amoskalev** [comment]
Can you tell what should I adjust in configs?

**Comment by amoskalev** [comment]
Hi @zyc-ai , can you help?

**Comment by amoskalev** [comment]
@zyc-ai @lb1100 @StrangerZhang Hi, how should I modify config to match 0.696 score on OTB?

**Reply by ZhiyuanChen** [reply]
It would take a lot of efforts to reproduce, just try.
